internal/service/cognitoidp/user_pool_test.go

```diff
@@ -1691,6 +1720,7 @@ resource "aws_cognito_user_pool" "test" {
   sms_configuration {
     external_id    = "test"
     sns_caller_arn = aws_iam_role.test.arn
+    sns_region     = "us-east-1"
```

Comment:
I'd avoid changing the existing tests as we need to ensure that there are no regressions when folks omit this optional attribute.

Reply:
Isn't that exactly why modifying the existing tests would be a good idea? It'll add this attr to them, hence ensuring that no regressions have happened? Happy to discuss if there's something I'm not quite getting.

Reply:
You'll want _at least_ one test that _doesn't_ provide the `sns_region` so that that use case is explicitly catered for, i.e. testing that everyone's current Terraform code is still handled correctly if they upgrade to a version of the provider that includes this new argument. Given all of the existing tests don't supply an `sns_region` currently (outside of this PR), that regression testing is already handled, so I'd leave those alone and just add add 1 new test case that _does_ supply it, which your PR already does.

Reply:
Ahh I see, ok. Thanks!